src/js/BuildApi.js

```diff
@@ -172,8 +172,8 @@ export default class BuildApi {
         this.load(url, onSuccess, onFailure);
     }
 
-    loadSponsorTile(onSuccess, onFailure) {
-        const url = `${this._url}/api/configurator/sponsors`;
+    loadSponsorTile(mode, onSuccess, onFailure) {
+        const url = `${this._url}/api/configurator/sponsors/${mode}`;
```

Comment:
Checking for `window.matchMedia('(prefers-color-scheme: dark)').matches` would do

Reply:
I'll test this.